src/main/java/core/mate/academy/service/BulldozerProducer.java

```diff
@@ -0,0 +1,17 @@
+package core.mate.academy.service;
+
+import core.mate.academy.model.Bulldozer;
+import core.mate.academy.model.Machine;
+import java.util.ArrayList;
+import java.util.List;
+
+public class BulldozerProducer implements MachineProducer {
+    @Override
+    public List<Machine> get() {
```

Comment:
```suggestion
    public List<Bulldozer> get() {
```
 ми ж працюємо з більдозерами, тому давай повертати їх. теж саме стосується решти MachineProducer